src/utils/string.js

```diff
@@ -29,3 +29,36 @@ export function removeURLs(
   );
   return string.replace(regExp, '$1' + redactedText);
 }
+
+/**
+ * Take an absolute file path string and sanitize it except the last file name segment.
+ *
+ * Note: Do not use this function if the string not only contains a file path but
+ * also contains more text. This function is intended to use only for path strings.
+ */
+export function removeFilePath(
+  filePath: string,
+  redactedText: string = '<PATH>'
+): string {
+  let pathSeparator = null;
+
+  // Figure out which separator the path uses.
+  if (filePath.includes('/')) {
+    // This is a Unix-like path.
+    pathSeparator = '/';
+  } else if (filePath.includes('\\')) {
+    // This is a Windows path.
+    pathSeparator = '\\';
+  }
+
+  if (pathSeparator === null) {
+    // There is no path separator, which means it's either not a file path or empty.
+    return filePath;
+  }
+
+  return (
+    redactedText +
+    pathSeparator +
+    filePath.substring(filePath.lastIndexOf(pathSeparator) + 1)
```

Comment:
optional: we tend to use `slice` instead of `substring` because the API is less confusing and similar to arrays. This is used the same way here. But up to you!